### PR TITLE
v12 cnv and fusion frequencies

### PR DESCRIPTION
**Pull request by zzgeng**
#### What scientific question is your analysis addressing?
Update cnv-frequencies and fusion-frequencies modules with v12 data and change the output column for `somatic_mutation` to `pediatric_cancer`

#### What was your approach?
Change `datatypeId` column value for the output table from `somatic_mutation` to `pediatric_cancer`

#### What GitHub issue does your pull request address?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/527
https://github.com/PediatricOpenTargets/ticket-tracker/issues/528

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/fusion-frequencies/results/.gitignore
Comment (kelseykeith) [inline analyses/fusion-frequencies/results/.gitignore:1]: I would suggest removing this `.gitignore` and pushing the results to GitHub. The `fusion-frequencies` files are small (< 10Mb) and well under GitHub's file size limit so we should be totally fine to continue hosting results here

**Review by kelseykeith — CHANGES_REQUESTED** [review]
Overall this looks great!

- Except for the PMTL column in the `cnv-frequencies` table (see Issues below), the structure of the new v12 tables for both modules match the v11 version.
- The data mostly match except for adjustments made with the new release.
- The `fusion-frequencies` module has been updated to use the DGD fusions in the `data/` directory, not the file inside the module
- The `datatypeId` column was correctly updated in both modules.

Minor Issues:
  - @ewafula, the PMTL annotation is being removed in v12 correct? So it's right that it's missing from the `cnv-frequencies` table when it was present in v11.
  - I suggested removing the `.gitignore` in the `fusion-frequencies` module since the files are all under 10Mb, well under GitHub's file size limit of 100Mb, but if we're moving away from this model it can be left alone

File: analyses/fusion-frequencies/results/.gitignore
Comment (zzgeng) [reply analyses/fusion-frequencies/results/.gitignore:1]: Thank you for your suggestion! I will update the `fusion-frequencies` results.

**Comment by ewafula** [comment]
> Overall this looks great!
> 
> * Except for the PMTL column in the `cnv-frequencies` table (see Issues below), the structure of the new v12 tables for both modules match the v11 version.
> * The data mostly match except for adjustments made with the new release.
> * The `fusion-frequencies` module has been updated to use the DGD fusions in the `data/` directory, not the file inside the module
> * The `datatypeId` column was correctly updated in both modules.
> 
> Minor Issues:
> 
> * @ewafula, the PMTL annotation is being removed in v12 correct? So it's right that it's missing from the `cnv-frequencies` table when it was present in v11.
> * I suggested removing the `.gitignore` in the `fusion-frequencies` module since the files are all under 10Mb, well under GitHub's file size limit of 100Mb, but if we're moving away from this model it can be left alone

@kelseykeith,  that is correct PMTL is no longer in all MTP tables. All MTP were all updated to exclude PMTL in a previous PR.

**Review by ewafula — APPROVED** [review]

**Review by kelseykeith — APPROVED** [review]

**Comment by kelseykeith** [comment]
Thanks for the changes @zzgeng! This looks good now, approved

**Pull request merged**